src/test/java/teammates/test/BaseTestCaseWithSqlDatabaseAccess.java

```diff
@@ -62,7 +63,17 @@ protected SqlDataBundle removeAndRestoreDataBundle(SqlDataBundle testData) {
      * Verifies that two entities are equal.
      */
     protected void verifyEquals(BaseEntity expected, ApiOutput actual) {
-        if (expected instanceof FeedbackQuestion) {
+        if (expected instanceof Student) {
+            Student expectedStudent = (Student) expected;
+            StudentData actualStudent = (StudentData) actual;
+            assertEquals(expectedStudent.getCourseId(), actualStudent.getCourseId());
+            assertEquals(expectedStudent.getEmail(), actualStudent.getEmail());
+            assertEquals(expectedStudent.getName(), actualStudent.getName());
+            assertEquals(expectedStudent.getComments(), actualStudent.getComments());
+            assertEquals(expectedStudent.isRegistered(), actualStudent.getJoinState() == JoinState.JOINED);
+            assertEquals(expectedStudent.getTeamName(), actualStudent.getTeamName());
+            assertEquals(expectedStudent.getSectionName(), actualStudent.getSectionName());
```

Comment:
with the merge with the master branch, there is another branch that has the same condition (expected instanceof student) in line 174, could I trouble you to remove that? thanks!

Reply:
I must have missed that when I rebased. I have removed mine.